tethysext/atcore/templates/atcore/resources/tabs/resource_list_tab.html

```diff
@@ -0,0 +1,65 @@
+<table class="table table-hover table-bottom-border" id="resource_table">
+            <thead>
+              {# NAME HEADER #}
+              <th>
+                <span>Name</span>
+              </th>
+
+              {# DESCRIPTION HEADER #}
+              <th>
+                <span>Description</span>
+              </th>
+
+              {# CREATED BY HEADER #}
+              <th>
+                <span>Created By</span>
+              </th>
+
+              {# DATE CREATED HEADER #}
+              <th>
+                <span>Date Created</span>
+              </th>
+
+              {# BUTTONS HEADER #}
+              <th></th>
+            </thead>
+            <tbody>
+              {% for resource in resources %}
+                <tr>
+                  {# NAME VALUE #}
+                  <td class="name" data-id="{{ resource.name }}" onclick="window.document.location='{{ resource.action_href }}';" style="cursor: pointer;">
+                    <a href="{{ resource.action_href }}">{{ resource.name }}</a>
+                  </td>
+
+                  {# DESCRIPTION VALUE #}
+                  <td class="name" data-id="{{ resource.name }}" onclick="window.document.location='{{ resource.action_href }}';" style="cursor: pointer;">
+                    {{ resource.description }}
+                  </td>
+
+                  {# CREATED BY VALUE #}
+                  {% if resource.created_by %}
+                  <td class="created_by" data-id="{{ resource.created_by }}" onclick="window.document.location='{{ resource.action_href }}';" style="cursor: pointer;">
+                    {{ resource.created_by }}
+                  </td>
+                  {% else %}
+                  <td class="created_by" data-id="Unknown" onclick="window.document.location='{{ resource.action_href }}';" style="cursor: pointer;">
+                    <i><font color="#A8A8A8">Unknown</font></i>
+                  </td>
+                  {% endif %}
+
+                  {# DATE CREATED VALUE #}
+                  <td class="date_created" data-id="{{ resource.date_created }}" onclick="window.document.location='{{ resource.action_href }}';" style="cursor: pointer;">
+                    {{ resource.date_created }} UTC
+                  </td>
+
+                  {# BUTTONS #}
+                  <td class="btn-td">
+                    <div class="btn-toolbar pull-right">
+                        <a class="btn btn-s btn-primary btn-flat btn-action btn-resource-launch" href="{{ resource.action_href }}" data-toggle="tooltip" data-placement="top" title="{{ resource.action_title }}"><span class="glyphicon glyphicon-chevron-right"></span></a>
+                    </div>
+                  </td>
+                </tr>
+              {% endfor %}
+            </tbody>
+          </table>
+        </div>
```

Comment:
Remove extra </div> and clean up indenting.

Reply:
don't know how that slipped through...